Website/DesktopModules/Hotcakes/Hotcakes.dnn

```diff
@@ -1174,6 +1174,47 @@
 			</components>
 		</package>
 
+		<package name="Hotcakes.MiniCartSkinObject" type="SkinObject" version="01.00.00">
```

Comment:
@ismaelTartola This version number should always match the version number in the rest of the HCC manifest.

Reply:
Changed to 03.07.00.